src/components/OnBoarding/FirstSwiper.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { styled } from "styled-components";
+import { MainLogo, TuticeTitleLogo } from "../../assets";
+
+export default function FirstSwiper() {
+  return (
+    <>
+      <FirstSwiperWrapper>
+        <TitleWrapper>
+          <FirstSwiperTitle>
+            쉬운 수업 관리로 열리는 <br /> 정확한 나의 <FirstSwiperTitleKeyword> 결실</FirstSwiperTitleKeyword>,
```

Comment:
이전 코드에서는 \n로 개행줬던데 여기선 <br />로 준이유가 잇나요?? 궁금해요~~~~~!!!!!!

> < br /> is a HTML line-break, whereas \n is a newline character in the source code.
> 
> In other words, <br /> will make a new line when you view the page as rendered HTML, whereas \n will make a new line when you view the source code.
> 
> Alternatively, if you're outputting to a console rather than somewhere that will be rendered by a web browser then \n will create a newline in the console output.